### DISCUSSION TITLE
建议halo增加python, golang等语言client

### DISCUSSION THREAD

**Discussion by hiBlunt** [halo-dev/halo#7133] [Ideas]
### Prerequisites

- [x] I have searched for related issues in the [Issues](https://github.com/halo-dev/halo/issues) list.
- [x] This is a feature related to Halo. If it is not an issue with the project itself, it is recommended to submit it in the [Discussions](https://github.com/halo-dev/halo/discussions).
- [x] If it is a feature suggestion for plugins and themes, please submit it in the respective plugin and theme repositories.

### Your current Halo version

2.20

### Describe this feature

目前halo api有ts client, 希望增加python， golang等语言client，方便大家使用。
PS：作为后端程序员实在不了解ts啊，直接用swagger文档中的api比较丑陋，只能自己用一用

**Comment by edops973** [comment]
I think it would be great to have client in Python and Golang
